.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.html]
+indent_style = space
+indent_size = 4
```

Comment:
It would be good to set up `prettier` to the pipeline to ensure this is the case. In pycroft, we suffer from an inconsistency in this regard: the base template uses 4 spaces, and all the other ones two spaces for indentation.

Reply:
(doing this in a separate branch, resolved)